samples/zigbee/ncp/src/main.c

```diff
@@ -103,6 +103,30 @@ int main(void)
 	(void)uart_line_ctrl_set(uart_dev, UART_LINE_CTRL_DCD, 1);
 	/* Data Set Ready - the NCP SoC is ready to communicate. */
 	(void)uart_line_ctrl_set(uart_dev, UART_LINE_CTRL_DSR, 1);
+
+#ifdef CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING
+	/* Configure USB device to use for logging ZBOSS Traces*/
+	uart_dev = device_get_binding(CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME);
```

Comment:
Consider moving the initialization to the backend sources instead of the `main.c` file.